qiling/loader/pe_uefi.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+#
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+# Built on top of Unicorn emulator (www.unicorn-engine.org)
+
+import ctypes
+import types
+import struct
+from unicorn import *
+from unicorn.x86_const import *
+from qiling.arch.x86_const import *
+from qiling.os.utils import *
+from qiling.const import *
+from qiling.os.memory import QlMemoryHeap
+from qiling.os.uefi.utils import *
+from qiling.os.uefi.uefi_types_64 import *
+from qiling.os.uefi.fncc import *
+from qiling.os.uefi.boot_services_hooks import *
+from qiling.os.uefi.runtime_services_hooks import *
+
+from qiling.os.windows.fncc import *
+from qiling.os.windows.fncc import _get_param_by_index
+
+import pefile
+from .loader import QlLoader
+
+class QlLoaderPE_UEFI(QlLoader):
+    def __init__(self, ql):
+        super()
+        self.ql = ql
+    
+    def run(self):    
+        self.ql.tpl = 4 # TPL_APPLICATION
+        self.ql.hook_override = {}
```

Comment:
we have set_api() method in Qiling class, so that you may use it instead of this hook_override ?